src/Contrib/Jaeger/SpanConverter.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Jaeger;
+
+use Jaeger\Thrift\Span as JTSpan;
+use Jaeger\Thrift\Tag;
+use Jaeger\Thrift\TagType;
+use OpenTelemetry\SDK\Trace\SpanDataInterface;
+
+class SpanConverter
+{
+    const STATUS_CODE_TAG_KEY = 'op.status_code';
+    const STATUS_DESCRIPTION_TAG_KEY = 'op.status_description';
+
+    /**
+    * Convert span to Jaeger Thrift Span format
+    */
+    public function convert(SpanDataInterface $span): JTSpan
+    {
+        $references = $tags = $logs = [];
+        $startTime = (int) ($span->getStartEpochNanos() / 1e3); // microseconds
+        $duration = (int) (($span->getEndEpochNanos() - $span->getStartEpochNanos()) / 1e3); // microseconds
+        $tags = [
+            self::STATUS_CODE_TAG_KEY => $span->getStatus()->getCode(),
+            self::STATUS_DESCRIPTION_TAG_KEY => $span->getStatus()->getDescription(),
+        ];
+
+        foreach ($span->getAttributes() as $k => $v) {
+            $tags[$k] = $this->sanitiseTagValue($v->getValue());
+        }
+        $tags = $this->buildTags($tags);
+
+        [
+            'traceIdLow' => $traceIdLow,
+            'traceIdHigh' => $traceIdHigh
+        ] = SpanConverter::convertOtlpToJaegerTraceIds($span->getContext()->getTraceID());
```

Comment:
can you make this `self::`, or even make it a non-static function and use `$this`

Reply:
Huh, did not know about self, neat! Much more readable